RELEASING.md

```diff
@@ -53,17 +55,27 @@ of the `cloud.google.com/go` repository root module. Note: releasing
 `cloud.google.com/go` has no impact on any of the submodules, and vice-versa.
 They are released entirely independently.
 
-# Test failures
+## Test failures
 
 If there are any test failures in the Kokoro build, releases are blocked until
 the failures have been resolved.
 
-# How to release `cloud.google.com/go`
+## How to release `cloud.google.com/go`
+
+Check for failures in the [continuous Kokoro build](http://go/google-cloud-go-continuous).
+If there are any failures in the most recent build, address them before
+proceeding with the release.
+
+### Automated Release
+
+If there are changes that have not yet been released a pull request should be
+automatically opened with a title like "chore: release 0.XX.0", where XX is the
+next version to be released. To cut a release approve and merge this pull
+request. Doing so will update the `CHANGES.md`, tag the merged commit with the
+appropriate version , and draft a GitHub release.
```

Comment:
Draft? Does the person doing the release need to do anything?

Reply:
I think the is the terminology used for releases. The button in Github to do this is `Draft a release`. And nope, it is done automagically. Here is an example of an automated github release: https://github.com/googleapis/google-cloud-go/releases/tag/v0.70.0